### PR TITLE
MS-22: Interviews - List interviews

### PR DESCRIPTION
**Pull request by NicoMunoz909**
Implemented 4 features:
- MS-19: Interviews - Create Interview
- MS-20: Interviews - Update Interviews
- MS-21: Interviews - Cancel Interview
- MS-22: Interviews - List Interviews

### CODE REVIEW

File: controllers/interviews.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        return req.params.id === interview.id;
+    });
+    if(index === -1) res.send('Could not find interview with specified ID');
+    for(property in req.query) {
+        data[index][property] = req.query[property];
+    }
+    res.send(`Interview succesfully updated! ${JSON.stringify(data[index])}`);
+}
+
+const cancel = (req, res) => {
```
Comment (damianalvarez) [inline controllers/interviews.js:29]: @NicoMunoz909 let's rename this method as `remove`

**Review by damianalvarez — CHANGES_REQUESTED** [review]

File: controllers/interviews.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        time: req.query.time,
+        status: req.query.status
+    }
+    data.push(interview);
+    res.send(`Interview succesfully created! ${JSON.stringify(interview)}`);
+}
+
+const update = (req,res) => {
+    const index = data.findIndex(interview => {
+        return req.params.id === interview.id;
```
Comment (EugenioPonzone) [inline controllers/interviews.js:20]: Muña,  you could take advantage of the arrow function and simplify in only one line without the return. 
Take a look on every case that you've implemented  arrow functions in that way.

File: controllers/interviews.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
+const fs = require('fs');
+const data = JSON.parse(fs.readFileSync('./data/interviews.json'));
+
+const create = (req,res) => {
+    const interview = {
+        id: `${Math.round(Math.random()*10000)}`,
```
Comment (BaltazarCortina) [inline controllers/interviews.js:6]: @NicoMunoz909 this way of creating ids can lead to repeated ids, we could solve this by checking that the new id doesn't already exist.

**Review by BaltazarCortina — CHANGES_REQUESTED** [review]

File: controllers/interviews.js (hunk 3 above)
Comment (NicoMunoz909) [reply controllers/interviews.js]: Didn't think about that. Fixed it using a do-while loop. Thanks!

**Review by damianalvarez — APPROVED** [review]

**Pull request merged**